cmd/shell.go

```diff
@@ -128,13 +141,26 @@ func shellRequestFromContext(currentContext utils.QoveryContext) (*pkg.ShellRequ
 		return nil, errors.New("Received " + res.Status + " response while fetching environment. ")
 	}
 
-	return &pkg.ShellRequest{
-		ApplicationID:  currentContext.ApplicationId,
-		ProjectID:      currentContext.ProjectId,
-		OrganizationID: currentContext.OrganizationId,
-		EnvironmentID:  currentContext.EnvironmentId,
-		ClusterID:      utils.Id(e.ClusterId),
-	}, nil
+	switch currentContext.ServiceType {
+	case utils.ApplicationType:
+		return &pkg.ShellRequest{
+			ApplicationID:  currentContext.ServiceId,
```

Comment:
The `ServiceId` shouldn't be set also now ? (I suppose the `ApplicationID`  field will be removed in the future ?)

Reply:
Indeed, it will be removed later, but for now we can't remove it